src/index.ts

```diff
@@ -245,6 +234,12 @@ function barfTurn() {
 
     // d. get dressed
     meatOutfit(embezzlerUp, [], underwater);
+    if (
```

Comment:
You prob don't need this here since the retrocape is being configured in meatOutfit